icons/build_icons.sh

```diff
@@ -134,6 +134,14 @@ build_win32() {
     convert -size 138x140 xc:white "${SRC_PREFIX}src/resources/win32/inno-small-250.bmp"
     composite -size 116x -gravity center icons/codium_only.svg "${SRC_PREFIX}src/resources/win32/inno-small-250.bmp" "${SRC_PREFIX}src/resources/win32/inno-small-250.bmp"
   fi
+  if [ ! -f "${SRC_PREFIX}build/windows/msi/resources/wix-banner.bmp" ]; then
```

Comment:
Both convert and wix (other than Inno) can use PNGs, can you check for an adjustment here to favor PNG over bmp?